sdb/commands/member.py

```diff
@@ -289,16 +289,17 @@ def _eval_member_terms(
             self._validate_type_dereference(obj, sep)
 
             #
-            # Ensure we are not trying to dereference NULL.
+            # Ensure we are not trying to dereference NULL or
+            # any kind of invalid memory.
             #
-            if obj.value_() == 0x0:
-                warning = f"cannot dereference NULL member of type '{obj.type_}'"
+            if not sdb.is_valid_memory_access(obj):
```

Comment:
I wonder if we should be letting drgn throw the memoy access exception and then catching it in the sdb infrastructure, outside of specific commands?

Reply:
I just issued https://github.com/delphix/sdb/issues/131 .